### ISSUE TITLE
Metadata lists and metadata delete

### ISSUE THREAD

**Issue by jurajplayercze** [SkriptLang/Skript#5160]
### Suggestion

Metadata are very useful and i like that. Pretty simple and easy to use.
I would love to be able to use something like metadata lists, working on same system like variable lists, where we can make "group" of metadatas. 
```
set metadata list "CustomList" of player to "test1" and "test2"
```

Also it would be awesome, if we were able to ```delete all metadatas of %metadataholders%``` at once. 

### Why?

It would add another flexible function for metadatas

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this suggestion.

**Comment by erenkarakal** [comment]
I'm pretty sure metadatas don't support multiple objects

**Comment by iPlexy** [comment]
yeah it doesnt as far as i know. It returns a list but can't be set to one

**Comment by TheLimeGlass** [comment]
As iPlexy said, it returns as a list. This could be added, but Spigot can't get all the metadata names of a holder.
